tests/unit/test_cocip_grid.py

```diff
@@ -324,7 +324,7 @@ def test_grid_survival_fraction(instance_params: dict[str, Any], source: MetData
         ),
```

Comment:
Probably worth adding a test for a `CocipGrid(met=met, rad=rad, dt_integration="1 hour")` case that would have failed before these changes.

Reply:
266176f79e9fa1bd2de325b55670d47f3f101513